tensorflow/compiler/mlir/hlo/include/mlir-hlo/Dialect/mhlo/transforms/fusion_utils.h

```diff
@@ -0,0 +1,194 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file seems like duplicated from https://github.com/tensorflow/tensorflow/pull/50020/files ?

Reply:
yes, fusion_utils depends on #50020 and will be removed after it's merged.

Reply:
It is merged, you can rebase

Reply:
done.